cs/src/core/ClientSession/ClientSession.cs

```diff
@@ -21,7 +21,7 @@ namespace FASTER.core
     /// <typeparam name="Output"></typeparam>
     /// <typeparam name="Context"></typeparam>
     /// <typeparam name="Functions"></typeparam>
-    public sealed class ClientSession<Key, Value, Input, Output, Context, Functions> : IClientSession, IDisposable
+    public sealed partial class ClientSession<Key, Value, Input, Output, Context, Functions> : IClientSession, IDisposable
```

Comment:
In FASTER.PSF.dll, we create an extension method on FASTER called `NewSessionWithPSF` that returns a `PSFClientSession`. This has a private field for `ClientSession`, and wraps the calls to Upsert etc. with PSF-specific crumbs. This takes care of the changes to ClientSession.cs